docs/developers_notes/04-regularizer.md

```diff
@@ -57,7 +57,7 @@ When developing a functional (i.e., concrete) `Regularizer` class:
 - **Must** define a default solver and a tuple of allowed solvers.
 - **May** require extra initialization parameters, like the `mask` argument of `GroupLasso`.
 
-!!! info
+??? tip "Convergence Test (Click to expand/collapse)"
```

Comment:
not sure you need the "click to expand/collapse". up to you, but I think it clutters it